js_modules/dagster-ui/packages/ui-core/src/assets/RecentUpdatesTimeline.tsx

```diff
@@ -251,7 +258,11 @@ const AssetUpdate = ({assetKey, event}: {assetKey: AssetKey; event: AssetEventTy
       case 'ObservationEvent':
         return <Icon name="observation" color={Colors.accentGreen()} size={16} />;
       case 'FailedToMaterializeEvent':
-        return <Icon name="run_failed" color={Colors.accentRed()} size={16} />;
+        return event.materializationFailureType === 'FAILED' ? (
+          <Icon name="run_failed" color={Colors.accentRed()} size={16} />
+        ) : (
+          <Icon name="status" color={Colors.accentGray()} size={16} />
+        );
     }
   }, [event.__typename]);
```

Comment:
I'm getting an error because i'm not including `event.materializationFailureType` in the list. but if i include `event.materializationFailureType` i get a different type error because some of the event types don't have that attribute. what's the right thing to do in this situation?

cc @hellendag